packages/cactus-plugin-ledger-connector-besu/src/main/typescript/plugin-ledger-connector-besu.ts

```diff
@@ -200,7 +202,7 @@ export class PluginLedgerConnectorBesu
       endpoints.push(endpoint);
     }
     {
-      const endpoint = new RunTransactionEndpoint({
+      const endpoint = new GetBlockEndpoint({
```

Comment:
@twiggins120: Oops, looks like here you deleted the RunTransactionEndpoint , if you add that back then it should be good

cc: @ty-lazar